taxjar-ruby.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 2.0'
 
   spec.add_dependency 'addressable', '~> 2.3'
-  spec.add_dependency 'http', '>= 1.0', '< 5.0'
+  spec.add_dependency 'http', '>= 4.4', '< 5.0'
```

Comment:
@milboj Per #57, could you change this to `spec.add_dependency 'http', '>= 4.3, '< 5.0'`?

Reply:
Thanks for bringing this to our attention @milboj. Since we want to get a release out for this today, I went ahead and updated the version. 👍